osu.Game/Storyboards/Drawables/DrawableStoryboardSample.cs

```diff
@@ -21,11 +21,29 @@ public class DrawableStoryboardSample : PausableSkinnableSound
 
         public override bool RemoveWhenNotAlive => false;
 
+        private readonly IBindable<bool> samplePlaybackDisabled;
+
         public DrawableStoryboardSample(StoryboardSampleInfo sampleInfo)
             : base(sampleInfo)
         {
             this.sampleInfo = sampleInfo;
             LifetimeStart = sampleInfo.StartTime;
+
+            samplePlaybackDisabled = SamplePlaybackDisabled.GetBoundCopy();
```

Comment:
Why does this make a private copy of a protected bindable owned by this exact same class?